h2/src/main/org/h2/util/ColumnNamer.java

```diff
@@ -124,26 +124,31 @@ private String generateUniqueName(String columnName) {
     }
 
     private boolean isAllowableColumnName(String proposedName) {
-
         // check null
         if (proposedName == null) {
             return false;
         }
         // check size limits
-        if (proposedName.length() > configuration.getMaxIdentiferLength() || proposedName.length() == 0) {
+        int length = proposedName.length();
```

Comment:
i assume we test for length() > 0 somewhere, since the Ignite regular expression means "1 or more" ?

Reply:
Such check is located below. It was here before my changes. And yes, old default pattern also had such requirement.